app/src/main/res/values/strings.xml

```diff
@@ -366,15 +366,15 @@
     <string name="sync_phase_apply_local_entries">reapplying local entries</string>
     <string name="sync_phase_push_entries">pushing entries</string>
     <string name="sync_phase_post_processing">post processing</string>
-    <string name="sync_error_unauthorized">Authentication failed</string>
-    <string name="sync_error_user_inactive">User is inactive</string>
+    <string name="sync_error_unauthorized">Could not log in</string>
```

Comment:
It's used for more than just login. It's also used for losing access (like the user inactive case). I think authentication failed is better.

Reply:
How about "Could not confirm identity"? Precisely because it is such a versatile word, it is hard to know exactly what it means and how to translate it.

Reply:
"Confirm identity" sounds even more vague. The problem is with the authentication and this title would be used with all authentication issues. It really is the authentication that failed.

I may be wrong, but the alternatives feel less clear to me than the original. I'm happy to add a "translator note" (how do you do it on Android?) if you think one is needed for translators.

Reply:
```
<!-- comment goes here -->
<string name="string_to_be_commented">Actual string to comment</string>
```
I removed "authorization" everywhere else, so hoping to make due without here too.

Reply:
How about something even more vague (and thus could be better shared): "Access denied"?

Reply:
Reply to https://github.com/etesync/android/pull/129#discussion_r494116166
I think "Access denied" is a good generic term that is easy to translate, no?

Reply:
Sure, it is far less complex.
"You were denied access" "Access was not granted" is how that would play out.
"App access denied" would remove all doubt, and it ties in a bit better with the login stuff where auth is removed elsewhere.

Reply:
Ahh, sorry, I misread the context, "Access denied" is not great, this string should strictly be about authentication. How about "Session expired"? Because this is what this error really means.

Reply:
OK, that has to be closer. Would "Your session expired and you need to log in again" fit?

Reply:
I think it's too long, it's just the short notification title. What's wrong with "Session expired"?

Reply:
It is a bit abrupt, but that is getting into fairly minor quibbles. :)

Reply:
I can live with "the session expired" :)